test/API/H5_api_attribute_test.c

```diff
@@ -101,9 +101,9 @@ test_create_attribute_on_root(void)
 
     TESTING_2("test setup");
 
-    if ((file_id = H5Fopen(H5_api_test_filename, H5F_ACC_RDWR, H5P_DEFAULT)) < 0) {
```

Comment:
`H5_API_TEST_FILENAME` is a macro that depending on build and test setup, is either the global name `H5_api_test_filename` or a threadlocal container name.